models/models.go

```diff
@@ -94,7 +95,22 @@ func Setup(c *config.Config) error {
 		return err
 	}
 	// Open our database connection
-	db, err = gorm.Open(conf.DBName, conf.DBPath)
+	max_attempts := 10
```

Comment:
I'd recommend moving this to a constant at the top of the file. Something like `MaxDatabaseConnectionAttempts`. This will make it more clear where this is coming from and easier to modify later on if needed.

Reply:
made it configurable from the config.json, default value will be 0 which mean: only 1 try